library/common-library/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: common-library
 description: Provides helpers to provide consistency on all the charts
 
 type: library
-version: 1.0.3
```

Comment:
at the end it is just changing a comment, a patch bump can be enough

Reply:
Sure. I am changing it before merging.